### DISCUSSION TITLE
The application won't start.

### DISCUSSION THREAD

**Discussion by edu8rio** [gitextensions/gitextensions#11407] [Q&A] [answered]
### Environment

 GitExtensions version: GitExtensions-4.2.0.17591-bf814d4f1.msi
- GIT version: git version 2.43.0.windows.1
- OS version: windows 11 22h2
- .NET version: dotnet --list-runtimes
    - Microsoft.WindowsDesktop.App 6.0.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    - Microsoft.WindowsDesktop.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    - Microsoft.WindowsDesktop.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    - Microsoft.WindowsDesktop.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

### Issue description

The application won't start.
It worked fine until a Windows update, I think.
There is no message, just double click and it doesn't work.
I have installed and uninstalled all 4.x versions.

### Steps to reproduce

Double-click and nothing happens.

### Did this work in previous version of GitExtensions?

Yes. First problem

**Comment by vbjay** [comment]
1. Reboot.  
1. Check event viewer for any issues

**Comment by edu8rio** [comment]
I have rebooted several times without success.

The event viewer, Windows logs, applications show an error: 

```xml
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
    <System>
    <Provider Name=".NET Runtime" /> 
    <EventID Qualifiers="0">1023</EventID> 
    <Version>0</Version> 
    <Level>2</Level> 
    <Task>0</Task> 
    <Opcode>0</Opcode> 
    <Keywords>0x80000000000000</Keywords> 
    <TimeCreated SystemTime="2023-11-28T16:25:16.6133842Z" /> 
    <EventRecordID>133251</EventRecordID> 
    <Correlation /> 
    <Execution ProcessID="16408" ThreadID="0" /> 
    <Channel>Application</Channel> 
    <Computer>[CENSORED]</Computer> 
    <Security /> 
    </System>
    <EventData>
        <Data>Description: A .NET application failed. Application: GitExtensions.exe Path: C:\Program Files (x86)\GitExtensions\GitExtensions.exe Message: Failed to create CoreCLR, HRESULT: 0x80004005</Data> 
    </EventData>
</Event>
```

[EDIT] fixed formatting

**Reply by RussKie** [reply]
This is the key problem:
```
Message: Failed to create CoreCLR, HRESULT: 0x80004005
```

Most generally speaking, error code 0x80004005 occurs when Windows cannot access a file or folder. You'll need to figure out what's causing that.

**Reply by edu8rio** [reply, marked as answer]
I don't know if this is the right solution, but I reinstalled .net (from the recommended [link](https://dotnet.microsoft.com/es-es/download/dotnet/thank-you/runtime-desktop-6.0.25-windows-x64-installer)) and everything worked for me.
I didn't even uninstall it, just used the repair option.

I've been using GitExtensions for a long time without any problems, so I don't know why this happened.
I'm sorry I can't be more specific and haven't tried this before.

Sorry for the inconvenience.

**Reply by vbjay** [reply]
Glad it worked. Prob like @RussKie said, maybe permissions of a file or file was deleted/renamed in update which broke .net. Also I have seen issues with start menu restart vs `shutdown -r` differences. I tend to use shutdown cli when I need a reboot from updates and such. Keep that in your toolbelt.

**Comment by RussKie** [comment]
No worries at all. Glad you got it working again.
